cg-spec/editors_draft.html

```diff
@@ -1858,50 +1858,46 @@ <h3>sameTerm</h3>
       </section>
       
       <section id="sparql-compare">
-        <h3>RDFstar-compare</h3>
+        <h3>sparql-compare</h3>
         
-        <p>The function <code>RDFstar-compare</code> is added in support of value-comparison
-          operators such as <code>=</code> and for <code>ORDER BY</code>.
+        <p>The function <code>sparql-compare</code> is defined to support of value-comparison
+          operators such as <code>=</code>, and for <code>ORDER BY</code>.
         </p>
         <p>
           <code>xsd:boolean</code> &nbsp;
-          <code style="color:black;font-weight:bold">RDFstar-compare</code>
-          (<code style="color:black">term</code>, <code style="color:black">term</code>)
+          <code style="color:black;font-weight:bold">sparql-compare</code>
+          (<code style="color:black">RDF-star term</code>,
+          <code style="color:black">RDF-star term</code>)
         </p>
-        <p>The function <code>RDFstar-compare</code> returns `-1`, `0`, `1`, or throws an error,
-          based on the comparison of the three elements of an RDF-star triple term.
+        <p>
+          The function <code>sparql-compare</code> returns `-1`, `0`, `1`, or throws an
+          error, based on the comparison of two
+          <a href="#dfn-rdf-star-terms">RDF-star terms</a>.
+          When comparing two RDF-star triple terms, comparison is 
+          by pairwise comparison of subject, then predicate, then object.
         </p>
         <div class="defn">
           <p>
             <b>Definition: 
               <dfn id="defn-sparql-compare">
-                <code style="color:black;font-weight:bold">sparql-compare</code></dfn></b>
+                <code style="color:black;font-weight:bold">sparql-compare</code></dfn>
+            </b>
           </p>
           <ul>
-            <li<h4>>If both A and B are not RDF-star triple terms,
-              compare by SPARQL 1.1 operators</h4>
-<code>&lt</code>, <code>=</code>, <code>&gt</code>)
-              and return that value (-1, 0, +1).<br/>
+            <li>If both A and B are not RDF-star triple terms,
```

Comment:
```suggestion
            <li>If neither A nor B is an RDF-star triple term,
```

Reply:
I, too, prefer this formulation,  it previous comments indicate that @afs is purposely avoiding neither/nor. Andy, can you explain your rationale?